lib/usb/usb_fx07_common.c

```diff
@@ -228,15 +228,22 @@ uint16_t stm32fx07_ep_read_packet(usbd_device *usbd_dev, uint8_t addr,
 	uint32_t extra;
 
```

Comment:
Previous: if one reads a unaligned small chunk, fifo may get out of sync and some data bytes omitted
Now: fifo stays in sync, but data bytes still omitted for second read.

If fifo gets out of sync, then even the basic functionality is broken.